docs/release-notes/8.4.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[release-notes-header-8.4.0]]
+== 8.4
+
+[discrete]
+[[release-notes-8.4.0]]
+=== 8.4.0
+
+[discrete]
+[[known-issue-8.4.0]]
+==== Known issue
+
+
+[discrete]
+[[breaking-changes-8.4.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: {pull}
+// end::breaking-changes[]
+
+[discrete]
+[[features-8.4.0]]
+==== Features
+* Creates a new rule type, New Terms, that fires when a value appears for the first time in a particular field ({pull}134526[#134526]).
+* Adds related alerts based on process ancestry to alert details ({pull}136009[#136009], {pull}138419[#138419], {pull}135340[#135340]).
+* Adds docs for Osquery API ({pull}137162[#137162]).
+* Creates the "Create dashboard" button to the Dashboards landing page ({pull}136671[#136671]).
+* Adds support for wildcard exceptions for detection rules. New operators are "matches" and "does not match" ({pull}136147[#136147]).
+* Adds a new search query parameter, `dry_run`, to the bulk actions API that allows you to simulate a bulk action without updating rules ({pull}134664[#134664]).
+* Creates the Responder, an interface that enables you to take actions on specific hosts ({pull}134520[#134520])
+* Enables a new method for the Task Manager API: `bulkUpdateSchedules`, which enables you to update the execution timing of `idle` tasks ({pull}132637[#132637]).
+
+[discrete]
+[[bug-fixes-8.4.0]]
+==== Bug fixes and enhancements
+* Updates the Network page's UI to match other overview pages ({pull}137541[#137541], {pull}136913[#136913]).
+* Fixes an error that could occur when you tried to apply an index pattern to rules as a bulk action ({pull}134664[#134664]).
+* Enhances rule previews with configurable rule intervals and look-back times ({pull}137102[#137102]).
+* Enhances the "status pending" badge for endpoint actions with detailed status when you hover on it ({pull}136966[#136966]).
+* Turns grouped navigation on by default ({pull}136819[#136819]).
+* Adds a confirmation dialog to bulk rule export ({pull}136418[#136418]).
+* Adds index pattern info to the Inspect panel({pull}136407[#136407]).
```

Comment:
```suggestion
* Adds index pattern information to the Inspect panel ({pull}136407[#136407]).
```

Reply:
Also, I'm not sure whether the window that appears is a panel or a some other UI element.